plugins/frontity-headtags/class-frontity-headtags-plugin.php

```diff
@@ -101,9 +102,13 @@ public function setup_integrations() {
 		if ( class_exists( 'WPSEO_Frontend' ) ) {
 			new Frontity_Headtags_Yoast();
 		}
-		// Setup All In One SEO Pack integration.
+		// Setup All In One SEO Pack integration (version 3).
 		if ( class_exists( 'All_in_One_SEO_Pack' ) ) {
-			new Frontity_Headtags_AIOSEO();
+			new Frontity_Headtags_AIOSEO_3();
+		}
+		// Setup All In One SEO Pack integration (version 4).
+		if ( function_exists( 'aioseo' ) ) {
```

Comment:
Might be a silly question because I don't know how things work in WP Plugin development. Is is possible that both `All_in_One_SEO_Pack` class and `aioseo` function exist? I guess it shouldn't be possible to install 2 versions of the same plugin, right?

Reply:
Nope, you cannot install 2 different versions of the same plugin. Internally, both versions would share the same plugin folder so one would overwrite the other.